test/Altinn.App.SourceGenerator.Tests/ModuleInitializer.cs

```diff
@@ -0,0 +1,13 @@
+using System.Runtime.CompilerServices;
+
+namespace Altinn.App.SourceGenerator.Tests;
+
+internal static class ModuleInitializer
+{
+    [ModuleInitializer]
+    public static void Init()
+    {
+        // VerifierSettings.AutoVerify(includeBuildServer: false);
```

Comment:
We have this pattern in the other initializers:

```csharp
        var isCi = !string.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable("CI"));
        if (BuildServerDetector.Detected && BuildServerDetector.IsWsl && !isCi)
            BuildServerDetector.Detected = false; // WSL is not a build server
        VerifierSettings.AutoVerify(includeBuildServer: false);
```

Reply:
Yes, fixed!